cmd/tikv-ctl/src/main.rs

```diff
@@ -796,7 +797,9 @@ fn print_bad_ssts(data_dir: &str, manifest: Option<&str>, pd_client: RpcClient,
         println!("\nsst meta:");
         // The output may like this:
         // --------------- Column family "write"  (ID 2) --------------
-        // 63:132906243[3555338 .. 3555338]['7A311B40EFCC2CB4C5911ECF3937D728DED26AE53FA5E61BE04F23F2BE54EACC73' seq:3555338, type:1 .. '7A313030302E25CD5F57252E' seq:3555338, type:1] at level 0
+        // 63:132906243[3555338 ..
```

Comment:
Bad format.

Reply:
yes, will fix it once we have a agreement on the comment width